### PR TITLE
test(#289): AT 289

### LINKED ISSUE #289: Acceptance Tests for #253 view all the documents and forms that have been submitted with a booking request
### User story
#253 

### Acceptance Test Checklist

- [x] View document/form by downloading it on booking request page.
- [x] Upload document/form on booking request page.

### PR DESCRIPTION
**Pull request by danseremet**
view all the documents and forms that have been submitted with a booking request

##### **_IMPORTANT: Always create an issue before a PR and provide enough information so that others can review_**

# AT #289 

This dusk test checks if the reference path that was saved is present in the Vue component.
The testing of opening files is already performed in feature tests.

##### _List all issues this PR resolves_
- closes #289

##### _Complete these items before requesting reviews_
- [x] Static Analysis Rules Validated
- [x] All commits follow naming conventions

### Sub-tasks Checklist
- [x] UI Implementation
- [x] UI Test
- [x] Integration Tests
- [x] System Tests
- [x] Documentation

### CODE REVIEW

**Review by EvanDime — APPROVED** [review]

File: tests/Browser/BookRoomsTest.php [hunk 1]
```diff
@@ -199,6 +200,7 @@ public function testPromptedToDownloadFormAndAttach()
                 ->type('#attendees', 3)
                 ->pause(250);
 
+            # Covers "Attach Reference Files #185"
             $browser->mouseover('#submit-booking-create')
                 ->check("#bake-sale-checkbox")
                 ->assertSee('Please remember to attach a filled out "Bake Sale Form" to the document section')
```
Comment (a-drew) [inline tests/Browser/BookRoomsTest.php:206]: This test needs to be expanded to cover the file attachment process

**Review by a-drew — CHANGES_REQUESTED** [review]
Test does not validate the attachment of files atm. It only validates that the error message shows up.

File: tests/Browser/BookRoomsTest.php (hunk 1 above)
Comment (a-drew) [reply tests/Browser/BookRoomsTest.php:206]: I'll also note that the `testPromptedToDownloadFormAndAttach()` test seems to cover this functionality already

**Comment by alexstojda** [comment]
Superseded by #677

**Pull request closed without merging**